SoapyRTLSDR.hpp

```diff
@@ -244,14 +266,15 @@ class SoapyRTLSDR: public SoapySDR::Device
     std::mutex _buf_mutex;
     std::condition_variable _buf_cond;
 
-    std::vector<std::vector<signed char> > _buffs;
+    std::vector<Buffer> _buffs;
     size_t	_buf_head;
     size_t	_buf_tail;
     std::atomic<size_t>	_buf_count;
     signed char *_currentBuff;
     std::atomic<bool> _overflowEvent;
     size_t _currentHandle;
     size_t bufferedElems;
+    long long bufTimeNs;
```

Comment:
I think it might be easier in the logic below to make this `bufTicks` to track the time in the remaining samples in readStream(). Just set bufTicks to _buffs[handle].tick in acquire(). Then it simplifies read stream:

```
timeNs = SoapySDR::ticksToTimeNs(bufTicks, sampleRate);
bufTicks += returnedElems; //for the next call to readStream if there is a remainder
```